generators/git/index.js

```diff
@@ -51,14 +51,8 @@ module.exports = generators.Base.extend({
   writing: function () {
     this.pkg = this.fs.readJSON(this.destinationPath(this.options.generateInto, 'package.json'), {});
 
-    var repository = '';
-    if (this.originUrl) {
-      repository = this.originUrl;
-    } else {
-      repository = this.options.githubAccount + '/' + this.options.name;
-    }
-
-    this.pkg.repository = this.pkg.repository || repository;
+    this.pkg.repository = this.pkg.repository || this.originUrl ||
```

Comment:
This is quite a long line. I'm all for simplicity, but that's not really simpler as is.